swagger.yaml

```diff
@@ -2049,6 +2049,14 @@ paths:
           schema:
             "$ref": "#/definitions/Spaces"
       parameters:
+      - in: query
+        name: filter
+        description: 'Returns a filtered list of `Collection`s. Valid options include
```

Comment:
You should be able to remove the outside quotes for easier reading (and not requiring the double `'`).

Reply:
This is autogenerated by a rake task that calls `to_yaml`. I guess that's just how it handles this situation.

Reply:
Ah, ok. That makes sense.